Tests/compat/sbs_func_args.py

```diff
@@ -32,6 +32,8 @@ def f3(arg0, arg1, arg2, *arg3): return "same## %s %s %s %s" % (arg0, arg1, arg2
     g3 = o.M3
     
     #for peverify runs
+    if clr.IsNetStandard:
```

Comment:
This would fail because it should be `clr.IsNetCoreApp`. Looks like ipy2 would need to be fixed as well. I'm guessing we don't run the sbs tests?

Reply:
I will verify, but I think we are running at least some of the sbs tests.

Reply:
See ironpython2/#250